README.md

```diff
@@ -105,7 +118,7 @@ or this [overview of the module's feature set](https://netatalk.io/docs/Webmin-M
 Bug reports and feature requests should be filed as [GitHub issue tickets](https://github.com/Netatalk/netatalk/issues).
 
 Before contributing code to the project, please read the [Developer Notes](https://netatalk.io/docs/Developer-Notes)
-to learn how to prepare a PR that is compliant with project guidelines.
+to learn coding standards and how to prepare a PR that is compliant with project guidelines.
```

Comment:
This reads a bit weird, could be read as 'it'll teach you how to code well' not 'what the conventions are'.  Perhaps "for the code and PR conventions used by this project" or something like that?  The verb 'to learn' feels quite wrong to me here.